client/gutenberg/extensions/map/component.js

```diff
@@ -254,7 +254,10 @@ export class Map extends Component {
 	};
 	loadMapLibraries() {
 		const { apiKey } = this.props;
-		import( /* webpackChunkName: "mapbox-gl" */ 'mapbox-gl' ).then( ( { default: mapboxgl } ) => {
+		Promise.all( [
+			import( /* webpackChunkName: "map/mapbox-gl" */ 'mapbox-gl' ),
+			import( /* webpackChunkName: "map/mapbox-gl" */ 'mapbox-gl/dist/mapbox-gl.css' ),
```

Comment:
Maybe we should have unique chunk names?

Reply:
If the files generated have different extensions, then the chunk is used only for filename. In this case, the build contains `map/mapbox-gl.css` and `map/mapbox-gl.js`. So it seems I could change the chunk name for clarity's sake, but it isn't necessary. In fact, this might even be the correct naming convention for script+style for the same library?

Reply:
Interesting, this actually puts the script and style into subdirectories. In a Jetpack site, we end up getting requests to:

```
https://cute-pilot-27.jurassic.ninja/wp-content/plugins/jetpack-dev/_inc/blocks/map/mapbox-gl.js
https://cute-pilot-27.jurassic.ninja/wp-content/plugins/jetpack-dev/_inc/blocks/map/mapbox-gl.css
```

Assets in the bundle now look like this:

```
npm notice
npm notice 📦  @automattic/jetpack-blocks@10.0.0
npm notice === Tarball Contents ===
npm notice 906B    package.json
npm notice 508B    README.md
npm notice 160B    build/block-manifest.json
npm notice 14.2kB  build/editor-beta.css
npm notice 211.5kB build/editor-beta.js
npm notice 14.2kB  build/editor-beta.rtl.css
npm notice 13.2kB  build/editor.css
npm notice 204.3kB build/editor.js
npm notice 13.2kB  build/editor.rtl.css
npm notice 57.8kB  build/images/cat-blog-87a988c0432f7f3ceb5bfe782db1431d.png
npm notice 26.6kB  build/images/devices-cddfc9159d108e7b62bcfbad49cdad37.jpg
npm notice 85.3kB  build/images/map-theme_black_and_white-1ead5946ca104d83676d6e3410e1d733.jpg
npm notice 113.5kB build/images/map-theme_default-2ceb449b599dbcbe2a90fead5a5f3824.jpg
npm notice 160.3kB build/images/map-theme_satellite-c74dc129bda9502fb0fb362bb627577e.jpg
npm notice 108.7kB build/images/map-theme_terrain-2b6e6c1c8d09cbdc58a4c0653be1a6e3.jpg
npm notice 38.2kB  build/images/mobile-wedding-ec7cce11cd6ea354451d78beac70755d.jpg
npm notice 1.2kB   build/images/oval-3cc7669d571aef4e12f34b349e42d390.svg
npm notice 7.5kB   build/images/paypal-button-1e53882e702881f8dfd958c141e65383.png
npm notice 8.2kB   build/images/paypal-button@2x-fe4d34770a47484f401cecbb892f8456.png
npm notice 22.4kB  build/map/mapbox-gl.css
npm notice 654.4kB build/map/mapbox-gl.js
npm notice 22.4kB  build/map/mapbox-gl.rtl.css
npm notice 406B    build/map/view.css
npm notice 19.3kB  build/map/view.js
npm notice 407B    build/map/view.rtl.css
npm notice === Tarball Details ===
npm notice name:          @automattic/jetpack-blocks
npm notice version:       10.0.0
npm notice package size:  917.3 kB
npm notice unpacked size: 1.8 MB
npm notice shasum:        ff9df7a874c15b6831481b709d7219bb89cbfd1e
npm notice integrity:     sha512-DNlB/ZsDxpXc4[...]Dh2fqCNfa/T5Q==
npm notice total files:   25
npm notice
```

This works well for the Jurassic Ninja builds, and just to confirm I copied the build output to a local Jetpack instance and it works

That looks good, today I learned 🙂

Reply:
The subdirectories are by design, because I used `map/mapbox-gl` as the chunk name. Initially I just had `mapbox-gl` which put the files at the root, but I thought they should live with their Map Block friends.